frontend/src/pages/SignPages/SignReceiver.tsx

```diff
@@ -0,0 +1,5 @@
+import FormikInContainer from 'components/SignForms/SignInForm/FormikInContainer';
+import SignPresenter from 'components/SignForms/SignPresenter/SignPresenter';
```

Comment:
Why do you use SignPages and SignForms naming if there is only one page and form?

Reply:
I renamed/deleted it